LayoutTests/webrtc/vp9-tests.js

```diff
@@ -30,14 +34,18 @@ test(() => {
         assert_equals(vp9Codecs[0].sdpFmtpLine, "profile-id=0", "first codec");
         assert_equals(vp9Codecs[1].sdpFmtpLine, "profile-id=2", "second codec");
     }
-
-    hasVP9 = vp9Codecs.length > 0;
 }, "VP9 in sender getCapabilities");
 
-test(() => {
+promise_test(async () => {
     if (window.internals)
         window.internals.setWebRTCVP9Support(false, false);
 
+    const videoConfiguration = { contentType: 'video/vp9', width: 800, height: 600, bitrate: 3000, framerate: 24 };
+    const results = await navigator.mediaCapabilities.decodingInfo({type: 'webrtc', video: videoConfiguration });
+    assert_false(results.supported, "decoder MC supported");
+    assert_not_equals(results.supportedConfiguration, undefined, "decoder MC configuration");
```

Comment:
supportedConfiguration isn't per spec anymore. We should be amending this

Reply:
I'll rebase the PR now that renaming PR landed.